ui_testing/pages/order_page.py

```diff
@@ -521,3 +521,28 @@ def remove_article(self, testplans=''):
             self.base_selenium.click(element='general:confirmation_button')
         self.sleep_small()
 
+    def set_random_test_unit(self, test_units='', sub_order_index=''):
+        suborder_table_rows = self.base_selenium.get_table_rows(
+            element='order:suborder_table')
+        suborder_row = suborder_table_rows[sub_order_index]
+        suborder_elements_dict = self.base_selenium.get_row_cells_id_dict_related_to_header(
+            row=suborder_row, table_element='order:suborder_table')
+        suborder_row.click()
+        if test_units:
+            self.base_selenium.select_item_from_drop_down(
+                element='order:test_unit', item_text=test_units)
+        else:
+            self.base_selenium.select_item_from_drop_down(
+                element='order:test_unit')
+            return self.get_random_test_unit()
+
+    def get_random_test_unit(self, sub_order_index =''):
```

Comment:
I found no need to define this new function, unless u need to get test unit of specific suborder

Reply:
done